src/core/registration.c

```diff
@@ -24,6 +24,12 @@
 #include "registration.c.clog.h"
 #endif
 
+typedef struct QUIC_REGISTRATION_EX {
+    QUIC_REGISTRATION Registration;
+    QUIC_REGISTRATION_CLOSE_COMPLETE_HANDLER CloseHandler;
+    void* CloseContext;
+} QUIC_REGISTRATION_EX;
```

Comment:
Instead of making a new EX struct. Just put this stuff inside the existing registration struct.